scripts/change-log-generator-utils.js

```diff
@@ -274,27 +274,17 @@ function generateKey(packageName, type, packagesToIgnore) {
 }
 
 function getReleaseDate() {
-  const dateArg = getArgumentValue(RELEASE_DATE_ARG);
-  return dateArg
-    ? dateArg
-    : new Intl.DateTimeFormat('en-US', {
-        month: 'long',
-        day: 'numeric',
-        year: 'numeric'
-      }).format(new Date());
+  // We want to ideally release two days from the day the release branch is cut
+  // (typically branch cut happens Monday and release on Wednesday)
+  let releaseDate = new Date();
+  releaseDate.setDate(releaseDate.getDate() + 2);
+  return new Intl.DateTimeFormat('en-US', {
+      month: 'long',
+      day: 'numeric',
+      year: 'numeric'
+    }).format(releaseDate);
```

Comment:
should we be removing the ability to override with the arg `RELEASE_DATE_ARG`?

Reply:
If we're ok with removing it, we can also remove the unused const RELEASE_DATE_ARG.

Reply:
Yes, this arg was passed in during changelog generation manually. We don't need this anymore.